doc/Sphinx/post-processing.rst

```diff
@@ -442,6 +444,9 @@ to manipulate the plotting options:
 
 * ``figure``: The figure number that is passed to matplotlib.
 * ``vmin``, ``vmax``: data value limits.
+* ``vsym``: indicates that data is symmetric about zero. In the absence
```

Comment:
I wonder whether it is possible to make a better parameter `vsym`. My point is that we need to set this parameter to `True` then choose a value for `vmax` if we want to set the value to a fixed number. What if `vsym` completely replaces `vmin` and `vmax` ? It could be set to `True` for auto limits, or to a float number for fixed limits `-vsym, vsym`. This is just an idea.

Anyways, the current documentation is a little confusing because it does not state that this parameter must be set to `True`. Furthermore, it does not explain the behavior when `vmax` and/or `vmin` are set to some value.

Reply:
Thanks for the suggestion, that's a really good idea. The way `vsym` currently works is that it only sets the data limits at all if both `vmin` and `vmax` are unset, with the reasoning being that if the user wants fixed symmetric limits it's easy to set that manually. I agree it would be a lot nicer to be able to set fixed limits directly in the `vsym` parameter, so I'll implement a change to allow this.

Do you have an idea of what the behaviour should be if `vsym` is set to a float and also one or both of `vmin`/`vmax` are set? I'd probably emit a warning and override the `vsym` limits with what's set in `vmin` and/or `vmax`, but if you think a different solution would be better I'll implement it that way.

Reply:
I somewhat believe `vsym` should take precedence as it is only a single instruction. This is not very important anyways. I would do something like:
```python
if vsym:
    if vmin or vmax:
        print("Some warning ...")
    if vsym is True:
        vmax = abs(A).max()
    else:
        vmax = abs(vsym)
    vmin = -vmax
```